### DISCUSSION TITLE
Trying to run from local llama.cpp server

### DISCUSSION THREAD

**Discussion by DefamationStation** [darrenburns/elia#67] [Questions]
Hello

I run a server at http://127.0.0.1:8080/v1 

and this is my config 

default_model = "Llama-3-8b-Q4_K_S"
system_prompt = ""
message_code_theme = "dracula"

[[models]]
name = "Llama-3-8b-Q4_K_S"
api_base = "http://127.0.0.1:8080/v1"
api_key = "api-key-if-required"

but I am getting this error

![image](https://github.com/darrenburns/elia/assets/82258900/ed4bb96c-c897-4bda-be2b-3f0e3b6209b4)

**Comment by jwinpbe** [comment]
I found elsewhere that you need to put `openai/` before the name of your model to get litellm to talk to llama.cpp and it worked for me.
